Tests iOS/MyList/MyListTests.swift

```diff
@@ -305,6 +305,25 @@ class MyListTests: XCTestCase {
         XCTAssertEqual(listView.itemCount, 3)
         XCTAssertEqual(listView.skeletonCellCount, 0)
     }
+
+    // MARK: - My List: Sort Items
+    func test_selectingANewSortOrder_SortItems() {
+        app.launch().tabBar.myListButton.wait().tap()
+        app.myListView.itemView(matching: "Item 1").wait()
+        
+        app.myListView.filterButton(for: "Sort/Filter").wait().tap()
+        app.sortMenu.sortOption("Oldest saved").wait().tap()
+        
+        app.myListView.itemView(matching: "Item 1").wait()
+        XCTAssertTrue(app.myListView.itemView(at: 0).contains(string: "Item 2"))
+        XCTAssertTrue(app.myListView.itemView(at: 1).contains(string:  "Item 1"))
+        
+        app.myListView.filterButton(for: "Sort/Filter").wait().tap()
+        app.sortMenu.sortOption("Newest saved").wait().tap()
+        
+        XCTAssertTrue(app.myListView.itemView(at: 0).contains(string: "Item 1"))
+        XCTAssertTrue(app.myListView.itemView(at: 1).contains(string:  "Item 2"))
+    }
```

Comment:
do we think this might be more suited for `MyListFiltersTests`? small note - not a blocker of any sort

Reply:
I'm okay either way but I feel 'MyListFiltersTests' are focused to filters only in terms of switching between segments (All, Fav). This is more like a feature that has a sorting impact on My List.

Reply:
Okay, just curious on which class would be better to put my Tags Filter work as well.

Reply:
iPad will have a follow up PR. For Tags Filter work, I'm not sure, sorry.

Reply:
@ckpocket the iPad work is now included in this PR, correct?

Reply:
@jacobsimeon yes it does.